games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -82,5 +82,28 @@ index aaa3a5b..03723fb 100644
    InitExitMessageFunction(DisplayExitMessage);
    InitExitFunction(CloseAllAndExit);
 -- 
-1.8.3.4
+2.7.0
```

Comment:
The first parts of the patch, I mean, those which we are not chaingin in this PR, were created by git 1.8.3.4, so there is no need to add noise to the commit. Keepin the original "1.8.3.4" will break anything.
Same thing for the SHA1 checksums. We can keep the original ones used for the first patches since we are not changing any of the first patches.